commands/screen.js

```diff
@@ -15,24 +17,31 @@ module.exports = async function (context) {
     return
   }
 
-  const domainQuestion = 'Add component to which domain?'
   const domains = filesystem.list('./src/views/')
   const domainChoices = ['(Create New)', ...domains]
+  let domainAddAnswer = {}
+  let domainPath = ''
+  if (!folder) {
+    const domainQuestion = 'Add screen to which domain?'
+    domainAddAnswer = await prompt.ask({
+      name: 'domain',
+      type: 'list',
+      message: domainQuestion,
+      choices: domainChoices
+    })
+    domainPath = (domainAddAnswer.domain === domainChoices[0]) ? '' : domainAddAnswer.domain + '/'
+  } else {
+    domainPath = (folder === 'views') ? '' : folder + '/'
+  }
```

Comment:
So important for this PR but I think we should abstract this domain handling code now that we have 3 generators essentially doing the same thing. 

Created a new issue for this here: https://github.com/infinitered/ignite-ir-boilerplate-bowser/issues/36

Reply:
Yeah, you're definitely right.